lib/osf-components/addon/components/search-result-card/template.hbs

```diff
@@ -49,18 +49,13 @@
         {{/if}}
         {{#if @result.isPartOf}}
             <div>
-                <span>{{t 'osf-components.search-result-card.from'}}: <a href={{@result.isPartOfTitleAndUrl.absoluteUrl}} target='_blank' rel='noopener noreferrer'> {{@result.isPartOfTitleAndUrl.title}} </a></span>
-            </div>
-        {{/if}}
-        {{#if @result.isContainedBy}}
-            <div>
-                <span>{{t 'osf-components.search-result-card.from'}}: <a href={{@result.isContainedByTitleAndUrl.absoluteUrl}} target='_blank' rel='noopener noreferrer'> {{@result.isContainedByTitleAndUrl.title}} </a></span>
+                <span>{{t 'osf-components.search-result-card.from'}}: <a href={{@result.isPartOf.absoluteUrl}} target='_blank' rel='noopener noreferrer'>{{@result.isPartOf.title}}</a></span>
```

Comment:
This diff doesn't look right. It should look like how it was before this change.

Reply:
Ok, let me check. I had accepted the upstream after checking GitHub's version.

Reply:
@adlius In the picture above, which version are you agreeing is correct?